app/views/directives/osc-secrets.html

```diff
@@ -1,50 +1,41 @@
+          {{displayType | startCase}} Secret
+        </label>
+      </div>
+      <div ng-repeat="pickedSecret in pickedSecrets">
+        <div class="secret-row">
+          <div class="secret-name">
             <ui-select ng-disabled="disableInput" ng-model="pickedSecret.name">
               <ui-select-match placeholder="Secret name">{{$select.selected}}</ui-select-match>
               <ui-select-choices repeat="secret in (secretsByType[type] | filter : $select.search)">
                 <div ng-bind-html="secret | highlight : $select.search"></div>
               </ui-select-choices>
             </ui-select>
           </div>
-          <div ng-if="allowMultipleSecrets">
-            <div class="basic-secrets">
-              <div class="secret-name">
-                <label ng-if="$first" class="picker-label">{{displayType | startCase}} Secrets</label>
-                <ui-select ng-disabled="disableInput" ng-model="pickedSecret.name">
-                  <ui-select-match placeholder="Secret name">{{$select.selected}}</ui-select-match>
-                  <ui-select-choices repeat="secret in (secretsByType[type] | filter : $select.search)">
-                    <div ng-bind-html="secret | highlight : $select.search"></div>
-                  </ui-select-choices>
-                </ui-select>
-              </div>
-              <div class="remove-secret">
-                <label class="sr-only">Remove Build Secret</label>
-                <a class="pficon pficon-close remove-btn" aria-label="Delete row" role="button" ng-click="removeSecret($index)" href=""></a>
-              </div>
-            </div>
-          </div>
-          <div ng-if="$last" ng-switch="displayType">
-            <div class="help-block" ng-switch-when="source">
-              Secret with credentials for pulling your source code.
-              <a href="{{'git_secret' | helpLink}}" target="_blank"><span class="learn-more-inline">Learn more&nbsp;<i class="fa fa-external-link" aria-hidden="true"></i></span></a>
-            </div>
-            <div class="help-block" ng-switch-when="pull">
-              Secret for authentication when pulling images from a secured registry.
-              <a href="{{'pull_secret' | helpLink}}" target="_blank"><span class="learn-more-inline">Learn more&nbsp;<i class="fa fa-external-link" aria-hidden="true"></i></span></a>
-            </div>
-            <div class="help-block" ng-switch-when="push">
-              Secret for authentication when pushing images to a secured registry.
-              <a href="{{'pull_secret' | helpLink}}" target="_blank"><span class="learn-more-inline">Learn more&nbsp;<i class="fa fa-external-link" aria-hidden="true"></i></span></a>
-            </div>
+          <div class="remove-secret">
+            <a ng-click="removeSecret($index)" class="pficon pficon-close remove-btn" aria-hidden="true" href="" role="button"></a>
+            <a class="sr-only">Remove build secret</a>
```

Comment:
To improve accessibility, this would be written like so:
```HTML
<a ng-click="removeSecret($index)"  href="" role="button" class="remove-btn">
  <span class="pficon pficon-close" aria-hidden="true"></span>
  <span class="sr-only">Remove build secret</span>
</a>
```

This hides the icon from screen readers, but keeps the descriptive text within the same element so the screen reader can properly describe the functionality of the link.